src/styles/index.scss

```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@import './untils/variables';
```

Comment:
Ensure that the path './untils/variables' is correct. It seems like there might be a typo in the directory name 'untils', which should likely be 'utils'. Verify the directory structure to prevent import errors.